etc/config/c++.amazon.properties

```diff
@@ -1132,6 +1132,35 @@ libs.tomlplusplus.versions.133.version=1.3.3
 libs.tomlplusplus.versions.133.path=/opt/compiler-explorer/libs/tomlplusplus/v1.3.3/include
 libs.tomlplusplus.versions.124.version=1.2.4
 libs.tomlplusplus.versions.124.path=/opt/compiler-explorer/libs/tomlplusplus/v1.2.4/include
+libs.trompeloeil.name=trompeloeil
+libs.trompeloeil.versions=40:39:38:37:36:35:34:33:32:31:30:29:28
+libs.trompeloeil.url=https://github.com/rollbear/trompeloeil
+libs.trompeloeil.versions.40.path=/opt/compiler-explorer/libs/trompeloeil/v40/include
+libs.trompeloeil.versions.40.version=v40
```

Comment:
The `v40` is the name that will appear in the dropdown - is this the "official" version you'd like users to see, or would you prefer it to appear as "40" ?

Reply:
v40 is the "official" name, if there is such a thing. It's the name referred to in the ChangeLog.

Reply:
Perfect! thanks!